### PR TITLE
Unit testing

### PR DESCRIPTION
**Pull request by Tim-Graupner**
Created unit tests for VehicleFormatter

### CODE REVIEW

**Review by jacob-briscoe — CHANGES_REQUESTED** [review]
Please refactor out the duplicate code for the templates as discussed. Thank you.

File: src/main/java/briscoe/cleancode/VehicleFormatter.java [hunk 1]
```diff
@@ -17,28 +29,22 @@ public String toMarkdown() {
...
-		html.append("<td>%1$s</td><td>%2$s</td><td>%3$s</td><td>%4$s</td><td>%5$s</td><td>%6$s</td><td>%7$s</td>\n");
-		html.append("</tr>\n");
-		html.append("</table>");
+		final String html = HTML_TEMPLATE;
+
 		return html.toString();
 	}
-	
+
 	private String markdownTemplate() {
```
Comment (jacob-briscoe) [inline src/main/java/briscoe/cleancode/VehicleFormatter.java]: You should use the new `MARKDOWN_TEMPLATE` variable in this method too. Right now it's duplicated still.

**Review by jacob-briscoe — CHANGES_REQUESTED** [review]
Much better, just finish fixing the duplicate markdown template string.

File: src/main/java/briscoe/cleancode/VehicleFormatter.java (hunk 1 above)
Comment (Tim-Graupner) [reply src/main/java/briscoe/cleancode/VehicleFormatter.java]: I thought I had, it's done now

**Review by jacob-briscoe — COMMENTED** [review]
What do you think about removing the `htmlTemplate` and `markdownTemplate` methods? They're not really doing anything now, just referencing a variable.

**Comment by Tim-Graupner** [comment]
> What do you think about removing the `htmlTemplate` and `markdownTemplate` methods? They're not really doing anything now, just referencing a variable.

I'll get rid of them and recommit.

**Review by jacob-briscoe — APPROVED** [review]
Nice job, this is much better. 👍

**Comment by jacob-briscoe** [comment]
Merge when you're ready.

**Pull request merged**